android/Smartbread/app/src/main/java/jp/hacks/smartbread/ui/main/pointcard/model/Point.kt

```diff
@@ -0,0 +1,8 @@
+package jp.hacks.smartbread.ui.main.pointcard.model
+
+import java.util.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jphacks/FK_1908/blob/3ccc72973dd6264abb8757f783aabaaca6b31bbc/android/Smartbread/app/src/main/java/jp/hacks/smartbread/ui/main/pointcard/model/Point.kt#L3"></span>Wildcard import (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>